changelog/release-6-3-4-2/2020-12-17-add-remove-constraint-property-method.md

```diff
@@ -0,0 +1,2 @@
+# Core
```

Comment:
Changelog is invalid Look into the ADR or other changelogs Also this file should be in _unreleased folder